### PR TITLE
Release 2021.02.01

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -6,9 +6,164 @@ services and the knowledge graph of Thoth.
...
+OpenShift cluster and automatically submits container images and build logs to Thoth Service. This bot helps Thoth
+to aggregate new knowledge about build failures and possible package issues.
+
+This is something that could even be deployed on you on premise cluster and help with extending the Thoth
+knowledge graph.
+
+### Update: Developer Chat
+
+As we can not open up the chat in general, we would like to ask you to open an issue on GitHub, and we will invite you
+to out channel. Please include the Google ID you would like us to send the invite to.
```
Comment (pacospace) [inline RELEASES.md:29]: `our`

File: RELEASES.md [hunk 2]
```diff
@@ -6,9 +6,164 @@ services and the knowledge graph of Thoth.
 Release cycles may vary from 2 to 4 weeks, depending on complexity of features or capacity of the team! For the current
 state of planning, have a look at our [Projects](https://github.com/orgs/thoth-station/projects).
 
+## Release 2021.02.01
```
Comment (fridex) [inline RELEASES.md:9]: We have also new configuration options propagated from .thoth.yaml. As they are not used in the recommendation engine itself, not sure if it is worth to be stated.

Comment (goern) [reply RELEASES.md:9]: lets keep them for a post when we can show them in action!

**Pull request merged**